src/acquisition/covidcast/csv_importer.py

```diff
@@ -224,7 +224,7 @@ def validate_missing_code(row, attr_quantity, attr_name):
     if hasattr(row, "missing_" + attr_name):
       missing_entry = getattr(row, "missing_" + attr_name)
       try:
-        missing_entry = int(missing_entry)
+        missing_entry = int(float(missing_entry)) # convert from string to float to int
```

Comment:
Would it make more sense to set the dtype for these when the file is read?

https://github.com/cmu-delphi/delphi-epidata/blob/cecb6c950e2e2f189d4b901557d9c54833eb85e6/src/acquisition/covidcast/csv_importer.py#L348

Reply:
Yea, but I think that would be a slightly larger refactor. A lot of `extract_and_check_row` is written around having everything be a string. I didn't want to go down that road right now, in favor of a quick fix.